apps/precursor-chemicals/fields/index.js

```diff
@@ -1,2 +1,56 @@
+const dateComponent = require('hof').components.date;
+
 module.exports = {
+  'responsible-officer-fullname': {
+    mixing: 'input-text',
+    validate: [ 'required' ],
+    className: ['govuk-input', 'govuk-!-width-two-thirds']
+  },
+  'responsible-officer-email': {
+    mixin: 'input-text',
+    validate: [ 'required', 'email' ],
+    className: ['govuk-input', 'govuk-!-width-two-thirds']
+  },
+  'responsible-officer-dbs-certificate': {
+    mixin: 'checkbox',
+    validate: [ 'required' ]
+  },
+  'responsible-officer-dbs-application-fullname': {
+    mixin: 'input-text',
+    validate: [ 'required' ],
+    className: ['govuk-input', 'govuk-!-width-two-thirds']
+  },
+  'responsible-officer-dbs-reference': {
```

Comment:
className: ['govuk-input', 'govuk-!-width-two-thirds'] - has to be added for this field to align with sigma